src/info.ts

```diff
@@ -0,0 +1,31 @@
+import { fileExists } from "./utils.js";
+import execute from "./spawn.js";
+
+export interface InfoSettings {
+  /** The path for the encrypted pdf */
+  input: string;
+  /** The password for the encrypted pdf */
+  password?: string;
+}
+
+/**
+ * Gets the encryption info for a PDF
+ * @param payload The settings for info
+ * @returns The output of QPDF
+ */
+export const info = async (payload: InfoSettings): Promise<Buffer> => {
+  if (!payload.input) throw new Error("Please specify input file");
+  if (!fileExists(payload.input)) throw new Error("Input file doesn't exist");
+
+  const callArguments = ["--show-encryption"];
+
+  // Password
+  if (payload.password) {
+    callArguments.push(`--password=${payload.password}`);
+  }
+
+  // Input file path
+  callArguments.push(payload.input);
+
+  return execute(callArguments);
```

Comment:
I think here it would be enough to extract the `:` params.

I have not tested the following snippet but it might do the trick:

```suggestion
  const outputParams = _.keyBy(
    (await execute(callArguments))
      .toString('utf8')
      .split('\n')
      .map(line => line.split(':').map(k => k.trim()).splice(0, 2)),
    p => p[0]
  );
  const isEncrypted = outputParams['Has user password'] === 'yes' || outputParams['Has owner password'] === 'yes';
  
  return { isEncrypted };
```